### PR TITLE
Fixed a minor grammatical mistake

### PR DESCRIPTION
**Pull request by begueradj**
Fixed a minor grammatical mistake.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Persona is not for everyone; if your login system is too complex and relies on m
 5. Allows changing passwords.
 6. Allows recovering forgotten passwords.
 
-## What does it NOT do?
+## What does NOT it do?
```
Comment (Kordonme) [inline README.md:23]: Didn't you type it wrong? It was correct before.

Comment (begueradj) [reply README.md]: Thanks for paying attention, but my version is the correct way. I can give you [this simple reference](https://www.englishgrammar.org/negative-questions/) for example. @kordonme

Comment (begueradj) [reply README.md:23]: Thanks for paying attention. It seems there are 2 ways to formulate negative questions and both the original version and mine are perfectly correct ([reference](https://www.englishgrammar.org/negative-questions/)). Hence I cancel my PR. Thank you again @kordonme

Comment (Kordonme) [reply README.md:23]: I don't disagree with you. It just sounds wrong. Maybe `What *DOESN'T it do?` is better then.

**Pull request closed without merging**